### PR TITLE
Enhancement/57/petra

### LINKED ISSUE #57: add PETRA to list of accelerators
- [x] PETRA accelerator class
- [ ] input file reader

optional:
- [ ] model creation for PETRA

### PR DESCRIPTION
**Pull request by awegsche**
And here is a shiny new accelerator.

Fixes #57

### CODE REVIEW

File: omc3/model/accelerators/petra/job.p3.madx [hunk 1]
```diff
@@ -0,0 +1,111 @@
+! --------------------------------------------------------------------------------------------------
```
Comment (JoschD) [inline omc3/model/accelerators/petra/job.p3.madx]: Maybe similar to LHC creation on the fly.. i.e. model creator? Less copy paste and more flexibility (tunes?)

File: omc3/utils/outliers.py [hunk 2]
```diff
@@ -41,6 +41,7 @@ def get_filter_mask(data, x_data=None, limit=0.0, niter=20, nsig=None, mask=None
         else:
             y, y_orig = _get_data_without_slope(mask, x_data, data)
         avg, std = _get_moments(y)
+
```
Comment (JoschD) [inline omc3/utils/outliers.py]: revert

File: omc3/optics_measurements/tune.py [hunk 3]
```diff
@@ -71,6 +74,7 @@ def phase_ac2bpm(self, df_idx_by_bpms, plane, accelerator):
         """
         model = accelerator.elements
         r = self.get_lambda(plane)
+        LOG.debug("lambda (plane {}) = {}".format(plane, r))
```
Comment (JoschD) [inline omc3/optics_measurements/tune.py]: f_strings

File: omc3/model/accelerators/petra.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+PETRA
+-------------------
+"""
+import os
+import datetime as dt
+from omc3.model.accelerators.accelerator import Accelerator, AcceleratorDefinitionError
+from omc3.model.constants import PLANE_TO_HV
+import logging
+
+LOGGER = logging.getLogger(__name__)
```
Comment (JoschD) [inline omc3/model/accelerators/petra.py]: not used

File: omc3/model/accelerators/petra.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+CURRENT_DIR = os.path.dirname(__file__)
```
Comment (JoschD) [inline omc3/model/accelerators/petra.py]: not used

File: omc3/model/accelerators/petra.py [hunk 6]
```diff
@@ -0,0 +1,29 @@
+"""
+PETRA
+-------------------
+"""
+import os
+import datetime as dt
```
Comment (JoschD) [inline omc3/model/accelerators/petra.py]: not used

File: omc3/model/manager.py [hunk 7]
```diff
@@ -4,9 +4,8 @@
 
 Contains entrypoint wrappers to get accelerator classes or their instances
 """
-from generic_parser.entrypoint_parser import entrypoint, EntryPoint, EntryPointParameters
-
-from omc3.model.accelerators import esrf, iota, lhc, ps, psbooster, skekb
+from generic_parser.entrypoint_parser import entrypoint, EntryPoint, EntryPointParameters, split_arguments
```
Comment (JoschD) [inline omc3/model/manager.py]: `split_arguments` is not used

File: omc3/optics_measurements/tune.py [hunk 8]
```diff
@@ -8,12 +8,14 @@
 Computes betatron tunes and provides structures to store them.
 """
 import numpy as np
-
-from omc3.optics_measurements.constants import PLANE_TO_NUM, PLANES
 from omc3.utils import stats
+from omc3.utils import logging_tools
+from omc3.optics_measurements.constants import PLANES, PLANE_TO_NUM
 
+LOG = logging_tools.get_logger(__name__)
```
Comment (JoschD) [inline omc3/optics_measurements/tune.py]: missing blank line

**Comment by JoschD** [comment]
Most of my comments are things that an IDE/VSCode would complain/solve automatically

File: omc3/model/accelerators/petra.py (hunk 6 above)
Comment (lmalina) [inline omc3/model/accelerators/petra.py]: Never used, remove

File: omc3/model/accelerators/petra.py (hunk 5 above)
Comment (lmalina) [inline omc3/model/accelerators/petra.py]: Never used, remove

File: omc3/model/manager.py (hunk 7 above)
Comment (lmalina) [inline omc3/model/manager.py]: split_arguments never used

File: omc3/model/accelerators/petra/job.p3.madx (hunk 1 above)
Comment (awegsche) [reply omc3/model/accelerators/petra/job.p3.madx]: That's how those files were created.

File: omc3/model/accelerators/petra.py (hunk 4 above)
Comment (awegsche) [reply omc3/model/accelerators/petra.py]: please remove

File: omc3/model/accelerators/petra.py (hunk 6 above)
Comment (awegsche) [reply omc3/model/accelerators/petra.py]: as discussed, model creation is the next step.

**Comment by fsoubelet** [comment]
If I may be the annoying one: please type hint the code? It makes a nice difference for anyone who hasn't written it.

**Review by lmalina — APPROVED** [review]

**Pull request merged**